templates/zerver/integrations.html

```diff
@@ -969,6 +969,33 @@
 
     </div>
 
+    <div id="gosquared" class="integration-instructions">
```

Comment:
`{{ external_api_uri_subdomain }}/v1/external/gosquared?api_key=abcdefgh&amp;stream=gosquared`

Was this the URL? You mean stream right? I couldn't find where I mentioned topic.